test/src/test/java/SslTest.java

```diff
@@ -0,0 +1,60 @@
+import io.restassured.RestAssured;
+import org.hamcrest.core.IsNull;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.*;
+
+
+public class SslTest extends AbstractFeatureServiceTest {
+
+    @Before
+    public void connectToSslPort() {
+        RestAssured.port = 8093;
+        RestAssured.baseURI = "http://localhost";
+    }
+
+    @Test
+    public void getServiceDescriptor() {
+        getRequest(request2path("featureService.json"))
+                .body("currentVersion", is(10.51f))
```

Comment:
Just do one assertion here - we don't care about all the data that's returned, we just want to make sure that an HTTPS call works (though I have been wondering that if ArcGIS requires HTTPS, perhaps all of our test servers should use HTTPS by default - that's out of scope for this PR , just something to consider).